native/src/widget/column.rs

```diff
@@ -10,9 +10,10 @@ use std::u32;
 
 /// A container that distributes its contents vertically.
 ///
-/// A [`Column`] will try to fill the horizontal space of its container.
+/// A [`Column`] will not fill the horizontal space of its container. (set to [`Length::shrink`] by defualt)
```

Comment:
Maybe we could simply remove the comment, as the `Shrink` behavior should be considered the default for all widgets.

Reply:
Yes, that's actually a good idea. Maybe we could document `Shrink` behaviour in the `widgets` module doc?

Reply:
Let's just remove the comments for now. I plan to review and improve all the documentation soon.

Reply:
Sure.